### DISCUSSION TITLE
Custom AR button's position shows different on iOS and Android

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1953] [Q&A]
Hello all,

As you can see in the tittle my custom ar button's position is different on ios and android.

There is picture in front of model-viewer on my page. When you click 3D button, images goes invisible. Other button which has AR icon is my custom AR button and it is out of model-viewers window via my css codes. So what could be problem here? any suggestion or solutions would be great.

ps: iOS version of the AR Button is correct position.

Thanks..

DEMO LINK
https://nervous-feynman-97da5e.netlify.app/urunrobot.html

MY CODES
https://jsfiddle.net/7fgvo1k3/

**Android SS**

![WhatsApp Image 2021-01-13 at 14 29 31](https://user-images.githubusercontent.com/68244546/104447316-aa9bb880-55ac-11eb-838e-0f11220b77f5.jpeg)

**iOS SS**

![IMG_1603](https://user-images.githubusercontent.com/68244546/104448354-1894af80-55ae-11eb-8c36-7679fbd928cb.PNG)

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x ] Safari

### OS

- [x ] Android
- [x ] iOS

**Comment by elalish** [comment]
This is just DOM, so it's a difference between how Chome and Safari handle CSS (Even Chrome on iOS is actually Safari). You might have even found a bug in how one of them handles slots.
